easybuild/easyconfigs/g/g2clib/g2clib-1.6.0-with-JesPer-2.0.14.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru g2clib-1.6.0-orig/enc_jpeg2000.c g2clib-1.6.0/enc_jpeg2000.c
```

Comment:
@SimonPinches Did you come up with this patch yourself?

If so, please add yourself as an another, and include a short description of the patch above this line.

If not, please include an upstream reference.

Reply:
Looks like this is based on http://forum.wrfforum.com/viewtopic.php?f=5&t=10248.

Unrelated, but confirming that just removing the use of `inmem_` is the correct fix: https://github.com/cloudruninc/wps/commit/a2fad80cc0ff8c1cba259f5aecc1b7393129cc00